lib/util/edit.ts

```diff
@@ -92,7 +93,7 @@ const createLanguageService = ({
   projectRoot: string;
   fileService: FileService;
 }) => {
-  const languageService = ts.createLanguageService({
+  return ts.createLanguageService({
```

Comment:
Tell me if you want me to revert these kind of changes, I've got some OCDs about this kind of stuffs 😬

Reply:
Reverted